rllib/policy/sample_batch.py

```diff
@@ -233,6 +235,8 @@ def __init__(self, *args, **kwargs):
         # Numpyfy seq_lens if list.
         elif isinstance(seq_lens_, list):
             self[SampleBatch.SEQ_LENS] = seq_lens_ = np.array(seq_lens_, dtype=np.int32)
+        elif (torch and torch.is_tensor(seq_lens_)) or (tf and tf.is_tensor(seq_lens_)):
+            self[SampleBatch.SEQ_LENS] = seq_lens_
```

Comment:
We need to allow sequence lengths as tensors here because we use them inside the learner to split batches where inputs are already converted to tensors.